installation_tips/README.md

```diff
@@ -1,36 +1,53 @@
 ## Installation tips
 
-If you are not (yet) an expert in Python installations (conda vs pip, mananging environements, etc.),
-here we propose a simple recipe to install `spikeinterface` and several sorters inside an anaconda
-environment for windows/mac users.
+If you are not (yet) an expert in Python installations, a main difficulty is choosing the installation procedure.
+The main ideas you need to know before starting:
+ * python itself can be distributed and installed many many ways.
+ * python itself do not contain so many features for scientifique computing you need to install "packages".
+   Numpy, scipy, matplotlib, spikeinterface, ... are python packages that have a complicated dependency graph between then. "uv"
+ * installing package  can be distributed and installed several ways (pip, conda, uv, mamba, ...)
+ * installing many packages at once is challenging (because of the depenency graph) so you need to do it in an "isolated environement"
+   to not destroy any previous installation. You need to see an "environement" as a sub installtion in dedicated folder.
+
+Choosing the installator + a environement manager + a package installer is a nightmare for beginners.
+The main options are:
+  * use "anaconda" that do everything. The most popular but bad idea because : ultra slow and agressive licensing (not free anymore)
```

Comment:
Maybe it is better to lead with the recommended one (i.e. uv first bullet point)?

On conda, is true in some cases but newer versions use `mamba` backend which is quite fast, and `conda-forge` does not have licensing issues. A problem is that HPC often come with the older slow versions and the default channel is anaconda's. 

No point going into too much detail but a suggestion to amend to say that that the default anaconda channel has agressive licensing and that it can be slow / is slower than uv. It is probably worth mentioning it can handle non-python dependencies while the other two can't, but this is not required for spikeinterface.

Reply:
I will try to modify a bit

Reply:
conda+mamba via https://github.com/conda-forge/miniforge is my favorite, doesn't have licensing problems, and almost required when setting up gpu dependencies